src/github_test.ts

```diff
@@ -14,6 +14,20 @@ Deno.test("getPrApprovers() returns the appropriate approvers", async () => {
   );
 });
 
+Deno.test("getPrApprovalNumber() returns the appropriate approval number", async () => {
+  const prToNumber = {
+    24270: 2,
+    24254: 2,
+    24259: 2,
+    24055: 0,
+  };
+  await Promise.all(
+    Object.entries(prToNumber).map(async ([pr, number]) => {
+      assertEquals(await getPrApprovalNumber(Number(pr)), number);
+    }),
+  );
+});
```

Comment:
It's a bit dangerous to test against PRs that could still change, but I guess if we lock those PRs, it should be safe enough.

Reply:
Those PRs are closed, I don't think they will change. If you can, lock them

Reply:
locked them